### ISSUE TITLE
EasyMode Config for Thermostat Devices

### ISSUE THREAD

**Issue by Flole998** [hobbyquaker/homematic-manager#100]
I request an easier approach to program the timing of a Max! thermostat, for example like on the Siemens Synco (https://www.tga-fachplaner.de/Archiv/Heft-Archiv/article-256568-100248/hlk-regler-synco-200-.html) in form of a table/graph. That makes it easy to keep an overview of how the times are programmed.

**Comment by hobbyquaker** [comment]
Max! is out of scope. This software only deals with homematic interface processes

**Comment by Flole998** [comment]
The devices are supported, working and shown, why not make them actually usable?

**Comment by Flole998** [comment]
Yes, when they are used with homegear they are shown, see attached screenshot.
![maxdevices](https://user-images.githubusercontent.com/9951871/50699696-db08fd80-1048-11e9-841e-f7004bde903f.PNG)

Also notice the horrible way of programming the timings....
![maxdevices1](https://user-images.githubusercontent.com/9951871/50699698-db08fd80-1048-11e9-80bf-f1bf330b1f2d.PNG)

**Comment by hobbyquaker** [comment]
aaah ok, now i got it :-) i‘m not using homegear, so i was confused.

**Comment by hobbyquaker** [comment]
Implemented in 3.0.0-beta.0 — including exactly your case.

The paramset editor now has device-specific editors sitting on top of the generic form, and one of them is a **heating week-programme editor**: weekday, time and temperature in a proper table instead of the raw `ENDTIME_*` / `TEMPERATURE_*` list from your second screenshot. It recognises four different naming shapes, and one of them is deliberately the **24-slot `TIMEOUT` / `TEMPERATUR` shape that a Max! thermostat answers with through Homegear** [#100]. A checkbox still shows the raw parameters when you want them.

To be honest about the scope: there is no Max!- or Homegear-specific code in 3.0 (deliberate decision — no vendor-specific paths). The editor keys off the paramset description the interface process reports, which is why your devices are covered anyway.

Beta: https://github.com/hobbyquaker/homematic-manager/releases/tag/v3.0.0-beta.0 — feedback on whether the editor recognises your thermostats would be genuinely useful; it was built against descriptions, not against a Max! device.

---
_Dieser Kommentar wurde von Claude, dem KI-Agenten des Maintainers, in dessen Auftrag geschrieben (hobbyquaker hat die Triage der offenen Issues zum 3.0-Beta-Release beauftragt). / This comment was written by Claude, the maintainer's AI agent, on his behalf as part of the issue triage for the 3.0 beta._
